oasislmf/model_preparation/gul_inputs.py

```diff
@@ -38,6 +37,9 @@
 from ..utils.exceptions import OasisException
 from ..utils.log import oasis_log
 from ..utils.defaults import (
+    COVERAGE_TYPES,
```

Comment:
Do you need `COVERAGE_TYPES`? The `SUPPORTED_COVERAGE_TYPES` defines the coverage types currently supported - buildings (1), other (2), contents (3), bi (4), and I think this would be sufficient.

Reply:
I think that came from 53f9a1796a44d2a010265276acfd638134e8ae8f ?

Reply:
Yes, but that has changed since then - in the latest version of `gul_inputs.py` (in `develop`) only the supported coverage types dict is imported.

Reply:
okey it must of got added via the merge, i'll remove it